Items/T3/Scepter.cs

```diff
@@ -260,7 +260,8 @@ private class ScepterReplacer {
         }
 
         private void Reroll(CharacterBody self, int count) {
-            if(count <= 0) return;
+            var bodyName = BodyCatalog.GetBodyName(self.bodyIndex);
+            if (count <= 0 || bodyName == "EngiWalkerTurretBody" || bodyName == "EngiTurretBody") return;
```

Comment:
Hardcoded strings like this may fail if another mod ever adds a new deployable. 2ab09f65e92a57af492fef85a3fc03789dbceec9: Check for the Deployable component instead, as that's something that should always be on deployables and only deployables.